### ISSUE TITLE
Crash Report: 1st instance of 10x crashed when I opened a 2nd

### ISSUE THREAD

**Issue by charlesastaylor** [slynch8/10x#1381]
Hi,

I've just uploaded a crash dump. I put my github username as the name field to match.

I had 10x open with an unreal engine solution. I opened a second instance of 10x, clicked "Create Workspace" and then the unreal engine solution instance crashed.

I've not tried to reproduce I'm afraid.

**Comment by slynch8** [comment]
Thanks for the crash dump. It looks like it's caused by changing the column count and then changing colour scheme. If the colour scheme was changed from the second instance it could make the first instance crash as it automoatically updates.

**Comment by slynch8** [comment]
Although I can't get it to happen. Let me know if you can get it again.

**Comment by charlesastaylor** [comment]
I'll give it a go tomorrow morning 👍

Not really any hassle when 10x opens up so fast 😀 I've used rider previously for unreal and no way I would even consider trying to track down bugs with 5+ minute open times. 10x also preserved the unsaved changes I had when it crashed, very noice.

**Comment by slynch8** [comment]
Thanks for trying. I tried your steps but couldn't get it either. Looking at the crash dump it's something to do with having an empty column and then changing the column count. The empty "filler" panel isn't getting removed from the list when the column is destroyed and then it crashes trying to update the colors on that filler panel. 

But I've tried every combination I can think of but still can't get it.
